requirements.txt

```diff
@@ -1 +1,2 @@
 Django==1.11.28
+pytz==2019.3
```

Comment:
I don't think this is needed because your project doesn't depend on this directly.